execute.go

```diff
@@ -70,16 +69,12 @@ func executeWithOutputChan(outputChan chan string, env []string, workingDir stri
 		return nil, fmt.Errorf("error starting process: %v", startErr)
 	}
 
-	waitErr := cmd.Wait()
+	//create a new effectiveconfig instance
+	var con = new(effectiveConfig)
```

Comment:
this is an incorrect way to create an effectiveConfig.  this will give you an empty one i believe.  If you need the effective config you should pass it in.  Personally as you really only need one thing off of the effective config (the build timeout) i'd just pass that in.